apps/admin/backend/src/app.test.ts

```diff
@@ -190,7 +190,7 @@ test('configuring with a CDF election', async () => {
   mockSystemAdministratorAuth(auth);
 
   const { electionData, electionHash } = safeParseElectionDefinition(
-    JSON.stringify(convertVxfElectionToCdfBallotDefinition(electionGeneral))
+    JSON.stringify(convertVxfElectionToCdfBallotDefinition(electionGeneral, {}))
```

Comment:
All of the consumers in this commit are either tests or scripts, so we can get by without actually providing translations